src/index.html

```diff
@@ -9,10 +9,56 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="styles/style.css"
     />
```

Comment:
The attributes of the <link> tag should start on a new line with 2-space indentation related to the tag, with the tag's closing bracket on the same level as the opening one.